plugins.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+var spawn = require('child_process').spawn;
+
+function InstallError(message, exitCode) {
+	Error.call(this, message);
+	this.exitCode = exitCode || 0;
+}
+
+InstallError.prototype = Object.create(Error.prototype);
+InstallError.prototype.constructor = Error;
+
+exports.InstallError = InstallError;
+
+exports.install = function (pluginName, cb) {
+	var opts = {
+		cwd: __dirname,
```

Comment:
I guess you have some misunderstanding. `__dirname` is where this file (`plugins.js`) is. Or, you explicitly set it to `cwd` for some reason?